.travis.yml

```diff
@@ -357,8 +358,8 @@ before_install:
 after_failure:
 # Disabled as otherwise the logfile is too big
 #  - if [ -f ${TRAVIS_BUILD_DIR}/config.log ]; then cat ${TRAVIS_BUILD_DIR}/config.log; fi
-  - if [ -f ${TRAVIS_BUILD_DIR}/ola-*/_build/config.log ]; then cat ${TRAVIS_BUILD_DIR}/ola-*/_build/config.log; fi
-  - if [ -f ${TRAVIS_BUILD_DIR}/ola-*/_build/sub/config.log ]; then cat ${TRAVIS_BUILD_DIR}/ola-*/_build/sub/config.log; fi
+#  - if [ -f ${TRAVIS_BUILD_DIR}/ola-*/_build/config.log ]; then cat ${TRAVIS_BUILD_DIR}/ola-*/_build/config.log; fi
+#  - if [ -f ${TRAVIS_BUILD_DIR}/ola-*/_build/sub/config.log ]; then cat ${TRAVIS_BUILD_DIR}/ola-*/_build/sub/config.log; fi
```

Comment:
Possibly put these back if you've got the debug you need as configure is obviously likely to fail first, although I guess test-suites fail more often so perhaps leave it.

I guess if we were being clever we'd set a variable when configure succeeded and use that to decide what diagnostic logs to show!

Reply:
Completely forgot I changed those....

Yeah, when someone has time...

Reply:
Actually, looking at yet another pair of random mac build failures, it occurred to me that just outputting the test-suite logs first would make sure they appear if the build failed in the test suite.  Seems like 99% of the time if it got that far, that's what you would want to see.

Reply:
Sometimes the simple fixes are the best!